tests/k8_dpdk_add_remove_services.py

```diff
@@ -0,0 +1,59 @@
+# limitations under the License.
+"""
+DPDK K8S: add/remove test pods with ping test
+"""
+
+# in-built module imports
+import time
+import sys
+
+# Unittest related imports
+import unittest
+
+# ptf related imports
+import ptf
+from ptf.base_tests import BaseTest
+from ptf.testutils import *
+from ptf import config
+
+# framework related imports
+import common.utils.log as log
+import common.utils.k8_utils as k8_utils
+
+
+class K8_DPDK_services(BaseTest):
+    def setUp(self):
+        BaseTest.setUp(self)
+        self.result = unittest.TestResult()
+        config[
+            "relax"
+        ] = True  # for verify_packets to ignore other packets received at the interface
+        test_params = test_params_get()
+        self.kube_iperf_path = test_params["kube_iperf_path"]
+
+    def runTest(self):
+        # Execute ./steps/setup.sh from https://github.com/Pharb/kubernetes-iperf3.git
```

Comment:
Do you assume that this repo is cloned before the start of this test?

Reply:
Will check, if not, will add another util to do this.